internal/cmd/environment/command_delete.go

```diff
@@ -45,5 +45,8 @@ func (c *command) delete(cmd *cobra.Command, args []string) error {
 	}
 
 	utils.ErrPrintf(cmd, errors.DeletedResourceMsg, resource.Environment, id)
+	if id == c.EnvironmentId() {
```

Comment:
We might want to remove stale env info from the context so that `c.EnvironmentId()` stops returning the deleted id as well in case someone doesn't immediately switch to a different one.

Reply:
i tried setting context auth account to nil, but it'll print out the `no org set` warning for every command you run until you set one. I think we didn't see this issue because we were always using the already deleted env id.

Reply:
Any idea how to fix this? I don't want to bring a warning for every following command, that might mess up users scripts

Reply:
> What should c.EnvironmentId() return if it shouldn't return the already deleted one?

It should definitely return `nil`!

> i tried setting context auth account to nil

This is the right way to do it, IMO. The warning (and possibly a few other things) just needs to be fixed.